tests/integration-tests/tests/networking/test_cluster_networking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import pytest
+from assertpy import assert_that
+
+
+@pytest.mark.dimensions("eu-west-1", "c4.xlarge", "alinux2", "slurm")
+@pytest.mark.dimensions("eu-west-2", "c5.xlarge", "alinux2", "sge")
+@pytest.mark.usefixtures("scheduler, os, region, test_datadir")
+def test_cluster_in_private_subnet(pcluster_config_reader, clusters_factory):
+    # This test just creates a cluster in the private subnet and just checks that no failures occur
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    assert_that(cluster).is_not_none()
```

Comment:
optional:
we could add a couple of assertions:
1. check that cloudformation stack is in create complete
2. check that the number of compute nodes is equal to 1. There is already a function that queries EC2 to get the cluster nodes.

Reply:
Shouldn't #1 be already tested from clusters_factpry?